.gitignore

```diff
@@ -178,6 +178,11 @@ cython_debug/
 
 # hydra directory
 outputs
+<<<<<<< 19-logging
```

Comment:
I think this is left over from a merge conflict, could you clear up the merge conflict annotations here, on line 184, and on line 188?

Reply:
resolved